x-pack/plugins/osquery/server/routes/privileges_check/privileges_check_route.ts

```diff
@@ -6,19 +6,28 @@
  */
 
 import type { IRouter } from '@kbn/core/server';
-import { OSQUERY_INTEGRATION_NAME, PLUGIN_ID } from '../../../common';
+import { OSQUERY_INTEGRATION_NAME } from '../../../common';
 import type { OsqueryAppContext } from '../../lib/osquery_app_context_services';
 
 export const privilegesCheckRoute = (router: IRouter, osqueryContext: OsqueryAppContext) => {
   router.get(
     {
       path: '/internal/osquery/privileges_check',
       validate: {},
-      options: {
-        tags: [`access:${PLUGIN_ID}-readLiveQueries`],
-      },
     },
     async (context, request, response) => {
+      // this is to skip validation eg. for analysts in cases attachments so they can see the results despite not having permissions
+      const { isSystemRequest } = request;
+      if (!isSystemRequest) {
```

Comment:
@elastic/kibana-security please keep me honest here. The system request header should NOT be used to skip authorization checks. The existence of the system request header is only intended to allow us to determine whether or not the user's session should be extended as a result of a HTTP request being made.

Reply:
Ok, the inner circle agreed to revert this changes and just show permission denied info. 
Thanks @kobelb for your input 👍

Reply:
Thanks for pinging us @kobelb! Yes, you're absolutely correct, `isSystemRequest == request that wasn't initiated by the user and shouldn't be treated as the user activity`.